src/JSInterop/Microsoft.JSInterop.JS/src/src/Microsoft.JSInterop.ts

```diff
@@ -66,7 +66,7 @@ module DotNet {
     }
   }
 
-  function invokePossibleInstanceMethodAsync<T>(assemblyName: string | null, methodIdentifier: string, dotNetObjectId: number | null, ...args: any[]): Promise<T> {
+  function invokePossibleInstanceMethodAsync<T>(assemblyName: string | null, methodIdentifier: string, dotNetObjectId: number | null, args: any[] | null): Promise<T> {
```

Comment:
I made this change as part of https://github.com/aspnet/Extensions/commit/d6bfc28e2104066dc363bf79bed39d6580977b26, but it breaks interop. Cannot use an args array as a parameter that accepts another args array